test/encode.cc

```diff
@@ -78,23 +78,36 @@ TEST(Encode, Polyline5) {
        {-78.55439, -25.28158},
        {-31.92992, 103.20477},
        {58.26482, -169.02219}},
-      "y|}~[fqox@jqrzEyjkm@~yfza@vmvgL}k~uXwkpcYrprzZ`ow~DisbzDrh{lSaebeCxqna@u~eoW}iq{Gnip|r@cdoeP");
+      "y|}~[fqox@jqrzEyjkm@~yfza@vmvgL_l~uXwkpcYtprzZ`ow~DisbzDrh{lSaebeCxqna@u~eoW}iq{Gnip|r@cdoeP");
 }
 
 TEST(Encode, Polyline) {
+
   /**
    * #to generate these a new test cases grab google polyline encoder for python:
    * wget "https://py-gpolyencode.googlecode.com/svn/trunk/python/gpolyencode.py"
    * sed -i -e "s/15E/16E/g" gpolyencode.py
    * #then generate a test case with python:
-   * python -c "import random; import gpolyencode; x = [ [round(random.random() * 180 - 90, 5),
-   * round(random.random() * 360 - 180, 5)] for a in range(0, random.randint(1,100)) ]; p =
-   * gpolyencode.GPolyEncoder().encode(x)['points']; print; print 'do_pair(' +
-   * str(x).replace('[','{').replace(']','}') + ', \"' + repr(p)[1:-1] + '\");'; print"
    */
 
+  // clang-format off
+/*
+
+python2 -c "
+import random;
+import gpolyencode;
+x = [ [round(random.random() * 180 - 90, 5), round(random.random() * 360 - 180, 5)] for a in range(0, random.randint(1,100)) ];
+p = gpolyencode.GPolyEncoder().encode(x)['points']; print;
+print 'do_polyline_pair(' + str(x).replace('[','{').replace(']','}') + ', \"' + repr(p)[1:-1] + '\");'; print"
```

Comment:
so this actually doesnt work because their encoder floors instead of rounds but its still useful for posterity (showing how we got these tests in the first place)